asv_bench/benchmarks/benchmarks.py

```diff
@@ -38,6 +38,8 @@
     trigger_import,
 )
 
+from modin.pandas import Series, Index, DataFrame, MultiIndex
```

Comment:
Need to use these objects from `IMPL` space.

Reply:
Removed this in latest commit.